<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite parallel="tests" name="Failed suite [Suite]" guice-stage="DEVELOPMENT" verbose="0">
  <listeners>
    <listener class-name="myparctice.testcomponents.Listeners"/>
  </listeners>
  <test thread-count="5" parallel="tests" name="Catch Errors(failed)" verbose="0">
    <classes>
      <class name="mypractice.tests.ErrorsHandingTest">
        <methods>
          <include name="LoginErrorCheck"/>
          <include name="navigateApp"/>
          <include name="closedriver"/>
        </methods>
      </class> <!-- mypractice.tests.ErrorsHandingTest -->
    </classes>
  </test> <!-- Catch Errors(failed) -->
  <test thread-count="5" parallel="tests" name="Submit Order(failed)" verbose="0">
    <classes>
      <class name="mypractice.tests.POMStandaloneTest">
        <methods>
          <include name="submitorder" invocation-numbers="0 1"/>
          <include name="OrderHistoryTest"/>
          <include name="navigateApp"/>
          <include name="closedriver"/>
        </methods>
      </class> <!-- mypractice.tests.POMStandaloneTest -->
    </classes>
  </test> <!-- Submit Order(failed) -->
</suite> <!-- Failed suite [Suite] -->
